docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -1142,12 +1142,22 @@ paths:
           maximum: 1000
         - name: offset
           in: query
-          description: page offset
+          description: page offset. Should be deprecated as may generate a lot of load for database.
           type: integer
           format: int64
           required: false
           default: 0
           minimum: 0
+        - name: token
+          in: query
+          description: |
+            Replacement for offset. Used as a base for detecting from where and in which direction to iterate over
```

Comment:
The point is that it should not be created and people should only use links from response

Reply:
I realized that the user should not be creating it only after I read the implementation.
It will be nice to start the description like 

> This is present in `next`/`prev` links in the response, and should not be generated by client.  Also it is not required for to get the very first request.

Maybe we can improve this in the next stage where there is no confusing "new" option.